core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -2011,6 +2015,12 @@ class WorkspaceService(
       })
       _ <- DBIO.from(Future.traverse(usersToInvite)(email => samDAO.inviteUser(email, ctx)))
 
+      _ <- traceDBIOWithParent("createTpsPao", parentContext) { span =>
+        DBIO.from(
+          policyServiceDAO.createWorkspacePao(UUID.fromString(workspaceId), workspaceRequest, span)
```

Comment:
Thinking out loud: I'm wondering if we should have a `PolicyService` intermediary between here and the policy DAO, to contain any business logic. For instance, the service class could inspect the workspaceRequest for its auth domain/protected data flag and build the appropriate policies.

As of this PR, it's really not necessary because the logic is pretty thin - so no changes requested, just something to keep in mind